packages/services/f-feature-management/src/evaluateRestriction.js

```diff
@@ -0,0 +1,220 @@
+import { logger } from './logger';
+
+/**
+ * Checks that the restriction object has at most one operator specified
+ * @param {object} restriction. Throws if not.
```

Comment:
I think this "Throws if not" is in the wrong place

Reply:
Updated